schemas/multidim-schema.json

```diff
@@ -740,222 +178,243 @@
+                "url"
+            ]
         },
-        "colorScale": {
+        "metadata": {
             "type": "object",
-            "description": "Color scale definition",
             "properties": {
-                "customNumericLabels": {
+                "name": {
+                    "type": "string"
+                },
+                "description": {
+                    "type": "string"
+                },
+                "descriptionShort": {
+                    "type": "string"
+                },
+                "descriptionFromProducer": {
+                    "type": "string"
+                },
+                "descriptionKey": {
                     "type": "array",
-                    "description": "Custom labels for each numeric bin. Only applied when strategy is `manual`.\n`null` falls back to default label.\n",
                     "items": {
-                        "type": [
-                            "string",
-                            "null"
-                        ]
+                        "type": "string"
                     }
                 },
-                "customCategoryColors": {
+                "descriptionProcessing": {
+                    "type": "string"
+                },
+                "unit": {
+                    "type": "string"
+                },
+                "shortUnit": {
+                    "type": "string"
+                },
+                "datasetName": {
+                    "type": "string"
+                },
+                "datasetId": {
+                    "type": "number"
+                },
+                "coverage": {
+                    "type": "string"
+                },
+                "nonRedistributable": {
+                    "type": "boolean"
+                },
+                "source": {
                     "type": "object",
-                    "description": "Map of categorical values to colors. Colors are CSS colors, usually in the form `#aa9944`",
-                    "patternProperties": {
-                        ".*": {
+                    "properties": {
+                        "id": {
+                            "type": "number"
+                        },
+                        "name": {
                             "type": "string"
+                        },
+                        "dataPublishedBy": {
+                            "type": "string"
+                        },
+                        "dataPublisherSource": {
+                            "type": "string"
+                        },
+                        "link": {
+                            "type": "string"
+                        },
+                        "retrievedDate": {
+                            "type": "string"
+                        },
+                        "additionalInfo": {
+                            "type": "string"
+                        }
+                    }
+                },
+                "origins": {
```

Comment:
We have some shared definitions in [definitions.json](https://github.com/owid/etl/blob/master/schemas/definitions.json#L144), so it should be possible to reuse it with
```
"origins": {
  "type": "array",
  "items": {
    "$ref": "definitions.json#/origin"
  }
}
```
If these origins are somehow different, then feel free to keep yours.

Reply:
I'd like to reuse the existing definition, however in the mdim config we expect the property names to be camel case, and the existing definition in ELT uses snake case. Do you know if/where does the conversion happen?

Reply:
> Do you know if/where does the conversion happen?

Hmm, that's a bummer. We switch from underscores to camelCase right before upserting objects (origins, indicators) to MySQL. I sincerely don't know if it's better to be consistent with the rest of ETL or with grapher.

Keep it as it is now, and we can revisit this at some point in the future.

Reply:
(I assume that Admin API validates field names, right?)

Reply:
Unfortunately, we do only minimal validation on the input in the admin, currently.

> We switch from underscores to camelCase right before upserting objects (origins, indicators) to MySQL.

This seems fine, and we could probably just do that everywhere. But the schema is (would be) a bit murky, since it's more about what we do in ETL and less about what the admin API expects. Which might be fine, I just assumed it's more of a contract between ETL and admin, and didn't realize it's mostly used as ETL internal DX affordance.